### PR TITLE
Pushing Negative Line and Areas Loads Result in positive Loads

### LINKED ISSUE #51: Pushing negative loads results in positive loads for GeometricalLineLoad and AreaUniformlyDistributedLoad
#### Description:
There are 2 related issues. This is the first.

Pushing negative loads for GeometricalLineLoad and AreaUniformlyDistributedLoad does not work. It always results in a positive value in RFEM6. 

#### Steps to reproduce:
Added test file

#### Expected behaviour:
Pushing actual negative values

#### Test file(s):
Test File: [link](https://burohappold.sharepoint.com/:u:/s/BHoM/ERuj0oXTGOtOnis8LuIdD08BaG97-V9bThGkTuaT5xsvTQ?e=bOxeAQ)

### LINKED ISSUE #52: Pushing x and y directional loads results in z-directional loads for GeometricalLineLoad
#### Description:
There are 2 related issues. This is the second.

Pushing negative loads for GeometricalLineLoad and AreaUniformlyDistributedLoad does not work. It always results in a positive value in RFEM6. 

#### Steps to reproduce:
Added test file

#### Expected behaviour:
Pushing actual negative values

#### Test file(s):
Test File: [link](https://burohappold.sharepoint.com/:u:/s/BHoM/ERuj0oXTGOtOnis8LuIdD08BaG97-V9bThGkTuaT5xsvTQ?e=bOxeAQ)

### PR DESCRIPTION
**Pull request by Arne-Martensen**
### Issues addressed by this PR

Initially, there have been some problems with pushing negative line loads and area loads. These issues should have been resolved with this PR.

Closes #51 
Closes #52 

This PR should resolve those above-mentioned issues.

### Test files

Test File: [Link](https://burohappold.sharepoint.com/:u:/s/BHoM/EckZJwISei1DqyXk6JTr5UQBrwSbfOd5-Tji29QVaYxD7A?e=lWfcno)

### Changelog

- Previously, the conversion of line loads from BHoM to RFEM6 was working using vector length, without taking the direction into account. This has been fixed.
- Fixed mismatching of Coordinate System under push process.

### CODE REVIEW

File: RFEM6_Adapter/Convert/ToRFEM6/BHoMDataStructure/Loading/Load.cs [hunk 1]
```diff
@@ -142,17 +142,21 @@ public static rfModel.surface_load ToRFEM6(this AreaUniformlyDistributedLoad bhA
 
             surface_load_load_direction loadDirecteion;
             Vector orientationVector = bhAreaLoad.Pressure;
+            double magnitude;
             if (orientationVector.X != 0)
             {
-                loadDirecteion = bhAreaLoad.Projected ? surface_load_load_direction.LOAD_DIRECTION_GLOBAL_X_OR_USER_DEFINED_U_PROJECTED : surface_load_load_direction.LOAD_DIRECTION_LOCAL_X;
```
Comment (HugoVanLooveren) [inline RFEM6_Adapter/Convert/ToRFEM6/BHoMDataStructure/Loading/Load.cs:147]: Would fix the names to loadDirection instead of typo loadDirecteion ;)

**Review by HugoVanLooveren — DISMISSED** [review]
Tested it with my own two scripts.
1. The AreaUniformlyDistributedLoad works with negative and positive values in the x, y, and z direction.
2. The geometrical line load only works in positive and negative z direction, not in x and y direction it seems.

File: RFEM6_Adapter/Convert/ToRFEM6/BHoMDataStructure/Loading/Load.cs (hunk 1 above)
Comment (Arne-Martensen) [reply RFEM6_Adapter/Convert/ToRFEM6/BHoMDataStructure/Loading/Load.cs:147]: I am assuming that you are referring to the push of free_line loads?

**Review by Kar-Myl — COMMENTED** [review]
The PR was tested using the provided test script as well as some user changes to the script.

Panel defined free line loads and Uniform area load were pushed in positive and negative directions. Loads were created as expected in RFEM6.

The code was not reviewed.

**Comment by Arne-Martensen** [comment]
@BHoMBot check compliance

**Comment by Arne-Martensen** [comment]
@BHoMBot check installer
@BHoMBot check core
@BHoMBot check null-handling
@BHoMBot check serialisation

**Review by FraserGreenroyd — APPROVED** [review]
Approving based on [this review](https://github.com/BHoM/RFEM6_Toolkit/pull/58#pullrequestreview-1867535518). Code not reviewed.

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**